# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1733858736 -3600
# Parent  1c96da2b61858a9bf653ad0e1e542e1a68313e43
Branding 6: Clean up About box, switch off the updater, fix donation links.
* * *
Misc: Add ! to helpus.end, to see get 'Make a donation!'
* * *
Branding: Add margin to about box.
* * *
Branding 6a: Move inline styles to CSS.

diff --git a/mail/base/content/aboutDialog.css b/mail/base/content/aboutDialog.css
--- a/mail/base/content/aboutDialog.css
+++ b/mail/base/content/aboutDialog.css
@@ -54,8 +54,12 @@ body {
     background-position: 100% 0;
   }
 }
 
+#versionBox {
+  margin-top: 10px;
+}
+
 #detailsBox {
   padding-top: 10px;
 }
 
@@ -92,8 +96,13 @@ body {
   margin-inline-start: 0;
   padding-inline-start: 0;
 }
 
+#betterbirdLink {
+  margin-bottom: 0;
+  font-size: 2em;
+}
+
 .update-deck-container {
   display: flex;
   align-items: center;
   position: fixed;
diff --git a/mail/base/content/aboutDialog.xhtml b/mail/base/content/aboutDialog.xhtml
--- a/mail/base/content/aboutDialog.xhtml
+++ b/mail/base/content/aboutDialog.xhtml
@@ -34,18 +34,16 @@
 #endif
   </xul:keyset>
   <div id="aboutDialogContainer">
     <xul:hbox id="clientBox">
-      <xul:vbox id="leftBox" flex="1"/>
-      <xul:vbox id="rightBox">
-        <xul:hbox align="baseline">
+      <xul:vbox id="leftBox" flex="0"/>
+      <xul:vbox id="rightBox" flex="1">
+        <xul:hbox id="versionBox" align="baseline">
           <span id="version"></span>
           <a id="releasenotes" class="text-link browser-link" hidden="hidden"
              data-l10n-id="release-notes-link"></a>
         </xul:hbox>
 
-        <img src="chrome://messenger/skin/icons/brand/eclipse-logo.webp" alt="" id="codenameLogo"/>
-
         <span id="distribution" class="text-blurb"></span>
         <span id="distributionId" class="text-blurb"></span>
 
         <xul:vbox id="detailsBox">
@@ -142,43 +140,25 @@
                data-l10n-attrs="{&quot;channel&quot;: &quot;&quot;}">
             <span id="currentChannel" data-l10n-name="current-channel"></span>
           </div>
 #endif
+          <!-- experimental is switched off via JS for release builds -->
           <xul:vbox id="experimental" hidden="true">
-            <div class="text-blurb" id="warningDesc">
-              <span data-l10n-id="warning-desc-version"></span>
-#ifdef MOZ_TELEMETRY_ON_BY_DEFAULT
-              <span data-l10n-id="warning-desc-telemetry"></span>
-#endif
-            </div>
-            <div class="text-blurb" id="communityExperimentalDesc" data-l10n-id="community-experimental">
-              <a class="text-link browser-link" href="https://www.mozilla.org/"
-                 data-l10n-name="community-exp-mozilla-link"></a>
-              <a class="text-link tab-link" href="about:credits"
-                 data-l10n-name="community-exp-credits-link"></a>
-            </div>
           </xul:vbox>
-          <div class="text-blurb" id="communityDesc" data-l10n-id="community-desc">
-            <a class="text-link browser-link" href="https://www.mozilla.org/" data-l10n-name="community-mozilla-link"></a>
-            <a class="text-link tab-link" href="about:credits" data-l10n-name="community-credits-link"></a>
+          <div id="betterbirdLink" class="text-blurb">
+            <a class="text-link browser-link" href="https://www.betterbird.eu/">www.betterbird.eu</a>
+          </div>
+          <!-- communityDesc is switched off via JS for Daily builds -->
+          <div class="text-blurb" id="communityDesc">
           </div>
           <div class="text-blurb" id="contributeDesc" data-l10n-id="about-donation">
-            <a class="text-link browser-link" href="https://www.thunderbird.net/donate/?utm_source=thunderbird-client&amp;utm_medium=referral&amp;utm_content=about-dialog"
+            <a class="text-link browser-link" href="https://www.betterbird.eu/donate/"
                data-l10n-name="helpus-donate-link"></a>
-            <a class="text-link browser-link" href="https://www.thunderbird.net/participate/"
+            <a class="text-link" href="mailto:betterbird@betterbird.eu?subject=Getting%20involved"
                data-l10n-name="helpus-get-involved-link"></a>
           </div>
         </xul:vbox>
       </xul:vbox>
     </xul:hbox>
-    <xul:vbox id="bottomBox">
-      <xul:hbox pack="center">
-        <a class="text-link bottom-link tab-link" href="about:license" data-l10n-id="bottom-links-license"></a>
-        <a class="text-link bottom-link tab-link" href="about:rights" data-l10n-id="bottom-links-rights"></a>
-        <a class="text-link bottom-link browser-link" href="https://www.mozilla.org/privacy/thunderbird/"
-           data-l10n-id="bottom-links-privacy"></a>
-      </xul:hbox>
-      <span id="trademark" data-l10n-id="trademarkInfo"></span>
-    </xul:vbox>
   </div>
 </body>
 </html>
diff --git a/mail/base/content/helpMenu.inc.xhtml b/mail/base/content/helpMenu.inc.xhtml
--- a/mail/base/content/helpMenu.inc.xhtml
+++ b/mail/base/content/helpMenu.inc.xhtml
@@ -5,30 +5,14 @@
 <menu id="helpMenu"
       data-l10n-id="menu-help-help-title"
       onpopupshowing="buildHelpMenu();">
         <menupopup id="menu_HelpPopup">
-          <menuitem id="menu_openHelp"
-                    data-l10n-id="menu-help-get-help"
-                    key="key_openHelp"
-                    oncommand="openSupportURL();"/>
-#if MOZ_UPDATE_CHANNEL == beta || defined(NIGHTLY_BUILD)
-          <menuitem id="menu_openReleaseHelp"
-                    data-l10n-id="menu-help-get-release-help"
-                    oncommand="openLinkText(event, 'releaseSupportURL');" />
-#endif
           <menuitem id="menu_keyboardShortcuts"
                     data-l10n-id="menu-help-shortcuts"
                     oncommand="openLinkText(event, 'keyboardShortcutsURL');"/>
-          <menuseparator/>
-          <menuitem id="getInvolved"
-                    data-l10n-id="menu-help-get-involved"
-                    oncommand="openLinkText(event, 'getInvolvedURL');"/>
           <menuitem id="donationsPage"
                     data-l10n-id="menu-help-donation"
                     oncommand="openLinkText(event, 'donateURL');"/>
-          <menuitem id="feedbackPage"
-                    data-l10n-id="menu-help-share-feedback"
-                    oncommand="openLinkText(event, 'feedbackURL');"/>
           <menuseparator id="functionsSeparator"/>
           <menuitem id="helpTroubleshootMode"
                     data-l10n-id="menu-help-enter-troubleshoot-mode"
                     oncommand="safeModeRestart();"/>
diff --git a/mail/base/content/utilityOverlay.js b/mail/base/content/utilityOverlay.js
--- a/mail/base/content/utilityOverlay.js
+++ b/mail/base/content/utilityOverlay.js
@@ -167,12 +167,9 @@ function openLinkText(event, what) {
     case "keyboardShortcutsURL":
       openUILink("https://support.mozilla.org/kb/keyboard-shortcuts/", event);
       break;
     case "donateURL":
-      openUILink(
-        "https://www.thunderbird.net/donate/?utm_source=thunderbird-client&utm_medium=referral&utm_content=help-menu",
-        event
-      );
+      openUILink("https://www.betterbird.eu/donate/", event);
       break;
     case "feedbackURL":
       openUILink("https://connect.mozilla.org/", event);
       break;
diff --git a/mail/branding/betterbird/content/aboutDialog.css b/mail/branding/betterbird/content/aboutDialog.css
--- a/mail/branding/betterbird/content/aboutDialog.css
+++ b/mail/branding/betterbird/content/aboutDialog.css
@@ -11,14 +11,16 @@
   background-repeat: no-repeat;
   /* min-width and min-height create room for the logo */
   min-width: 210px;
   min-height: 210px;
-  margin-top: 20px;
+  margin-top: 15px;
+  margin-bottom: 15px;
   margin-inline-start: 30px;
 }
 
 #rightBox {
   background-size: 260px auto;
+  margin-top: 35px;
 }
 
 #codenameLogo {
   height: 28px;
diff --git a/mail/components/customizableui/content/panelUI.inc.xhtml b/mail/components/customizableui/content/panelUI.inc.xhtml
--- a/mail/components/customizableui/content/panelUI.inc.xhtml
+++ b/mail/components/customizableui/content/panelUI.inc.xhtml
@@ -564,36 +564,16 @@
     <panelview id="appMenu-helpView"
                data-l10n-id="appmenu-help-panel-title"
                class="PanelUI-subView">
       <vbox class="panel-subview-body">
-        <toolbarbutton id="appmenu_openHelp"
-                       class="subviewbutton subviewbutton-iconic"
-                       data-l10n-id="appmenu-help-get-help"
-                       key="key_openHelp"
-                       oncommand="openSupportURL();"/>
-#if MOZ_UPDATE_CHANNEL == beta || defined(NIGHTLY_BUILD)
-        <toolbarbutton id="appmenu_openReleaseHelp"
-                       class="subviewbutton subviewbutton-iconic"
-                       data-l10n-id="appmenu-help-get-release-help"
-                       oncommand="openLinkText(event, 'releaseSupportURL');" />
-#endif
         <toolbarbutton id="appmenu_keyboardShortcuts"
                        class="subviewbutton subviewbutton-iconic"
                        data-l10n-id="appmenu-help-shortcuts"
                        oncommand="openLinkText(event, 'keyboardShortcutsURL');"/>
-        <toolbarseparator/>
-        <toolbarbutton id="appmenu_getInvolved"
-                       class="subviewbutton subviewbutton-iconic"
-                       data-l10n-id="appmenu-help-get-involved"
-                       oncommand="openLinkText(event, 'getInvolvedURL');"/>
         <toolbarbutton id="appmenu_makeDonation"
                        class="subviewbutton subviewbutton-iconic"
                        data-l10n-id="appmenu-help-donation"
                        oncommand="openLinkText(event, 'donateURL');"/>
-        <toolbarbutton id="appmenu_submitFeedback"
-                       class="subviewbutton subviewbutton-iconic"
-                       data-l10n-id="appmenu-help-share-feedback"
-                       oncommand="openLinkText(event, 'feedbackURL');"/>
         <toolbarseparator/>
         <toolbarbutton id="appmenu_troubleshootMode"
                        class="subviewbutton subviewbutton-iconic"
                        data-l10n-id="appmenu-help-enter-troubleshoot-mode2"
